sage-ios/SAGE/SAGE/AnnouncementsOperations.swift

```diff
@@ -11,11 +11,16 @@ import AFNetworking
 
 class AnnouncementsOperations {
     
-    static func loadAnnouncements(filter filter: [String: AnyObject]? = nil, completion: (([Announcement]) -> Void), failure:((String) -> Void)) {
+    static func loadAnnouncements(page: Int? = nil, filter: [String: AnyObject]? = nil, completion: (([Announcement]) -> Void), failure:((String) -> Void)) {
```

Comment:
make it page page: Int?= nil so you need to write page manually when you call the method